src/AutorestSwift/View Models/ResponseViewModel.swift

```diff
@@ -33,14 +33,17 @@ enum ResponseBodyType: String {
     case body
     /// Service returns no response data, only a status code
     case noBody
+    case stringBody
+    case intBody
+    case jsonBody
```

Comment:
What is the difference between `body` and the three other body types?

Reply:
these 3 body types just dedicate how we decode the message body. 
* jsonBody - use JsonDecoder
* string - decode with String.init
* int - decode with String.init and then covert to int

Reply:
But where do we actually determine the right one and set it? i only see in the init where we choose between body, pagedBody and noBody, which implies to me that we are overloading this param in a way we shouldn't.

Reply:
It is set in ExceptionResponseViewModel 
![image](https://user-images.githubusercontent.com/8701731/93144517-0574e000-f69f-11ea-80c6-5843f7d05463.png)

Reply:
Ah, then I think it would make more sense, instead of adding those values to ResponseBodyType, to make a new enum ExceptionResponseBodyType, that has these values. Otherwise, one enum is describing two different things, which could make future maintenance more difficult.

Reply:
Github swallowed my comment, so I'll repost. :)  

It seems this should be a different enum, `ExceptionResponseBodyStrategy` instead of being folded into `ResponseBodyStrategy` since the two sets of values are mutually exclusive. For maintainability, combining them could be confusing (i.e. the difference between .body and .bodyJson?)

Reply:
Good point. Thanks.